packages/web/components/cards/generic-main-card.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { ReactNode } from "react";
+
+import { IconLink } from "~/components/cards/icon-link";
+
+export const GenericMainCard: React.FC<{
+  children: ReactNode;
+  title?: string;
+  titleIcon?: ReactNode;
+  titleIconAction?: string;
+  width?: string;
+}> = ({ children, title, titleIcon, titleIconAction, width = "27" }) => {
+  return (
+    <div
+      className={`relative flex flex-col gap-8 overflow-hidden rounded-[24px] bg-osmoverse-800 px-1 py-1 lg:mx-auto md:mt-mobile-header md:gap-6 md:px-3 md:pt-4 md:pb-4`}
```

Comment:
Also, throughout this component, we should see if we could use the default spacing instead of using custom pixel values. Only use if absolutely necessary.

<img width="475" alt="Screenshot 2023-07-29 at 7 25 57 AM" src="https://github.com/osmosis-labs/osmosis-frontend/assets/4606373/34abede0-2b80-4ea3-9101-7912fd092683">